ocaml/typing/typecore.ml

```diff
@@ -840,8 +881,7 @@ let has_mode_annotation annots annot =
     (fun x -> Jane_syntax.N_ary_functions.mode_annotation_equal x.txt annot)
     annots
 
-let mode_annots_none =
-  {locality = None; uniqueness = None; linearity = None}
+let mode_annots_none = (None, None, None)
```

Comment:
This was previously a record with names. That seems better than just a tuple -- especially as the number of mode axes increases.

Reply:
Good point - done.